src/Compilers/CSharp/Test/Symbol/Symbols/Source/RecordTests.cs

```diff
@@ -1176,6 +1176,21 @@ .maxstack  2
 }");
         }
 
+        [Fact]
+        public void PartialTypes_04_PartialBeforeModifiers()
+        {
+            var src = @"
+partial public record C
+{
+}
+";
+            CreateCompilation(src).VerifyDiagnostics(
+                // (2,1): error CS0267: The 'partial' modifier can only appear immediately before 'class', 'struct', 'interface', or 'void'
```

Comment:
>before 'class', 'struct', 'interface', or 'void' [](start = 91, length = 48)

Is this list adjusted in another PR?

Reply:
Yes. I'm merged master into this PR and updated the comment now.